editor/animation_track_editor.cpp

```diff
@@ -5074,7 +5074,8 @@ void AnimationTrackEditor::_fetch_value_track_options(const NodePath &p_path, An
 		case Variant::PACKED_FLOAT64_ARRAY:
 		case Variant::PACKED_VECTOR2_ARRAY:
 		case Variant::PACKED_VECTOR3_ARRAY:
-		case Variant::PACKED_COLOR_ARRAY: {
+		case Variant::PACKED_COLOR_ARRAY:
+		case Variant::PACKED_VECTOR4_ARRAY: {
```

Comment:
Added this in the latest push.

Might be worth checking if other changes are required for animations to fully support the new type @TokageItLab 
In a quick test animating a PackedVector4Array property seems to work with AnimationPlayer and continuous mode.